### DISCUSSION TITLE
"convert convert" in error messages

### DISCUSSION THREAD

**Discussion by jidanni** [ImageMagick/ImageMagick#6626] [General]
Are these double names still present in the latest version?
$ convert /dev/null
convert convert: Request did not return an image.

**Comment by urban-warrior** [comment]
You are not running the ImageMagick `convert` utility.  ImageMagick's `convert` does not support your exception message.  Perhaps you are running a different `convert` utility or perhaps a shell script.

**Comment by jidanni** [comment]
OK, thanks. I filed
https://sourceforge.net/p/graphicsmagick/bugs/727/
